dev/lib/helpers.sh

```diff
@@ -31,31 +31,35 @@ function waitForPod() {
     MINUTE=0
     podName=$1
     ignore=$2
-    running="\([0-9]\+\)\/\1"
-    printf "${BLUE}Waiting for ${podName} to reach running state (4min).\n${CLEAR}"
```

Comment:
Removed the colours as they weren't used in `start.sh` nor in `dev/start.sh`.
Please double-check if this change is okay.

Reply:
I actually added the coloration (hopefully with a toggle to turn them off) in brew-start.sh in hopes that we would get some cycles to properly re-touch start.sh and dev/start.sh to clean them up a bit.  Admittedly coloration isn't the cleanest thing - so I'm _definitely_ open to feedback - do you find it disruptive/obtrusive.  I'm not opposed to wholesale removing it.  

I also now see that you've done the heavy lifting of removing the baked-in versions of the helpers files I broke out (but importantly didn't change references - we were late in the release and didn't want to risk destabilizing CI).

Reply:
Thanks!